linz-data-importer/linz_data_importer.py

```diff
@@ -305,6 +305,11 @@ def initGui(self):  # pylint:disable=invalid-name
         item.setIcon(QIcon(image_path))
         self.dlg.uListOptions.addItem(item)
 
+        item = QListWidgetItem("About")
+        image_path = os.path.join(os.path.dirname(__file__), "icons", "about.png")
+        item.setIcon(QIcon(image_path))
+        self.dlg.uListOptions.addItem(item)
```

Comment:
There are now six repeated blocks like this; it could be pulled out into a function to create a `QListWidgetItem` from just a title and an image name (technically you could pass in just the title and then use `title.lower()` for the icon name, but that's kinda magic and not very flexible). You could also pull out `icons_path = os.path.join(os.path.dirname(__file__), "icons")`.